lib/data/model/movies_model.dart

```diff
@@ -0,0 +1,39 @@
+import '../../domain/entity/movie_entity.dart';
+
+class MoviesModel extends MovieEntity {
```

Comment:
Same as before, this model represents a single `Movie`

Reply:
Done!